sdk-setup/src/rpmvalidation.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Copyright (C) 2013 - 2014 Jolla Ltd.
+# Contact: Ville Nummela <ville.nummela@jolla.com>
+#
+# This file is part of the SailfishOS SDK
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+usage() {
+    cat <<EOF
+Execute RPM validators for the given RPM file
+
+Usage: 
+    $0 -t <target> <rpm-file>
+	
+Options:
+    -t|--target <target> Execute the validators on target.
+EOF
+}
+
+run_validators() {
+    VALIDATORS=$(sb2 -t $OPT_TARGET ls /usr/libexec/validators)
+    echo "Using validators: $VALIDATORS"
+	for f in $VALIDATORS; do
+		sb2 -t $OPT_TARGET "/usr/libexec/validators/$f" $@ || break
+	done
+}
+
+while [[ $1 ]]; do
+    case "$1" in
```

Comment:
No need to quote. Basic rule: quote every argument to a "command" (unless word splitting is wanted), do not quote any argument to shell language constructs. Check e.g. sdk-motd for reference.